jetbrains-core/src/software/aws/toolkits/jetbrains/utils/SdkWaiterUtils.kt

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.utils
+
+import software.amazon.awssdk.core.waiters.WaiterResponse
+
+fun <T> WaiterResponse<T>.response(): T {
+    val responseOrException = this.matched()
+    if (responseOrException.response().isPresent) {
+        return responseOrException.response().get()
+    }
+    if (responseOrException.exception().isPresent) {
```

Comment:
Should we check exception first? (is it possible to have both?)

Reply:
It is not according to matt